### PR TITLE
feat: quality-of-life improvements for profile

### PR DESCRIPTION
**Pull request by ashmidgley**
[Develop Improvements to Profile](https://rational-nomads.notion.site/Develop-Improvements-to-Profile-f67ea7fb5f7449f09654c4c257b00e3e)

-----

## 1. Pull request checklist
- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
```
- Feature
```

## 3. What is the old behaviour?
- The profile page requires some quality-of-life improvements to provide a better user experience.

## 4. What is the new behaviour?
 - Multiple updates to 'My Domains' and 'My Bids' to provide better user experience (See requirements in Notion card).
 - Changes to semantics/styling of back navigation in profile.
 - Changes to make profile language consistent. Mixed usage of ‘My’, ‘Owned’ and ‘Your’ has been changed to just 'My' where applicable.

### CODE REVIEW

File: src/components/Artwork/Artwork.module.scss [hunk 1]
```diff
@@ -28,9 +28,70 @@
...
+	}
+
+	.Arrow {
+		height: 1px;
+		width: 8px;
+		position: absolute;
+		left: 0;
+		top: 50%;
+		background: #{colors.$grey-lighter-2};
+	}
```
Comment (colbr) [inline src/components/Artwork/Artwork.module.scss:72]: Can we use `scss` nesting here, e.g.

```
.ArrowContainer {
    &:hover {...}
    .Arrow { ... }
}
```

File: src/components/Cards/ImageCard/ImageCard.module.scss [hunk 2]
```diff
@@ -7,20 +7,17 @@ $placeholder-background: #000;
...
-
 	position: relative !important;
-
 	background: #{colors.$background-primary};
 	border: 1px solid #{colors.$mine-shaft};
 	border-radius: calc(#{borders.$radius-primary} / 2) !important;
-
 	cursor: pointer;
-	overflow: hidden;
+	overflow: visible;
```
Comment (colbr) [inline src/components/Cards/ImageCard/ImageCard.module.scss:15]: This `overflow` is causing the design feedback I gave (image canvas escaping the border).

File: src/containers/Tables/BidTable/BidTableRow.test.tsx [hunk 3]
```diff
@@ -32,7 +30,7 @@ jest.mock(
 );
 
 const renderComponent = ({ data = mockData } = {}) => {
-	return render(
+	return renderWithRedux(
```
Comment (colbr) [inline src/containers/Tables/BidTable/BidTableRow.test.tsx:33]: Does this test need to use `renderWithRedux`? I don't think BidTableRow is using redux, so can probably just use `render`.

File: src/containers/Tables/BidTable/BidTableRow.tsx [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 
 //- Library Imports
 import { ethers } from 'ethers';
 import { Domain } from 'lib/types';
 
 //- Constants Imports
-import { TOKEN } from './BidTableRow.constants';
+import { getNetworkZNA } from 'lib/utils';
+import { useHistory } from 'react-router-dom';
+import { ROUTES } from 'constants/routes';
```
Comment (colbr) [inline src/containers/Tables/BidTable/BidTableRow.tsx:17]: nit: can we rearrange these imports a little - `getNetworkZNA` under `Library imports`, etc.

**Review by colbr — CHANGES_REQUESTED** [review]
Some code changes and some design changes. Looking good though, almost there!

File: src/containers/Tables/BidTable/BidTableRow.test.tsx (hunk 3 above)
Comment (ashmidgley) [reply src/containers/Tables/BidTable/BidTableRow.test.tsx:33]: After introducing row row click navigation to BidTableRow, the existing test fails due to undefined useHistory(). I Used `renderWithRedux` in this case for Router setup.

I could manually wrap this test with `<Router>` but figured cleaner to just use `renderWithRedux`. Thoughts?

File: src/containers/Tables/BidTable/BidTableRow.tsx (hunk 4 above)
Comment (ashmidgley) [reply src/containers/Tables/BidTable/BidTableRow.tsx]: Good call, have addressed in latest commit!

File: src/components/Cards/ImageCard/ImageCard.module.scss (hunk 2 above)
Comment (ashmidgley) [reply src/components/Cards/ImageCard/ImageCard.module.scss:15]: Good spotting. Have made change to address this in latest commit.

File: src/components/Artwork/Artwork.module.scss (hunk 1 above)
Comment (ashmidgley) [reply src/components/Artwork/Artwork.module.scss]: Good call. Have fixed up in latest commit.

File: src/containers/Tables/BidTable/BidTableRow.test.tsx (hunk 3 above)
Comment (colbr) [reply src/containers/Tables/BidTable/BidTableRow.test.tsx:33]: Makes sense! `renderWithRedux` is generally slower to test with than just `render` - alternatively you could use `render` and mock `useHistory`. I think `renderWithRedux` is all good for now though, as we'll be rewriting these tests in the not-so-distant future.

**Pull request closed without merging**